joomla/joomla-cms
# [4.0] Move component specific fields out of com_menus

Pull Request for Issue # .

### Summary of Changes

Moves component-specific field class out of `com_menus` and into respective components.
Also corrects class/filename casing.

### Testing Instructions

Create administrator menu without preset.

Go to create a new menu item.

Select `Categories -> Create New Category` and `Categories -> List All Categories` menu item types. In their forms check that `Choose a Component` dropdown appears.

Select `Fields -> List All Field Groups`, `Fields -> List All Fields` and `Workflows -> List All Workflows` menu item types. In their forms check that `Context` dropdown appears.

### Expected result

Works like before.

### Documentation Changes Required

IDK.
